lintConfig.xml

```diff
@@ -5,21 +5,21 @@
     <ignore path="**-iw/**.xml" />
   </issue>
   <issue id="LintError">
-    <ignore regexp=".*BookmarksRobot.kt.*"/>
-    <ignore regexp=".*DebugFunctions.kt.*"/>
-    <ignore regexp=".*HistoryRobot.kt.*"/>
-    <ignore regexp=".*IntroRobot.kt.*"/>
-    <ignore regexp=".*LanguageRobot.kt.*"/>
-    <ignore regexp=".*LibraryRobot.kt.*"/>
-    <ignore regexp=".*LocalFileTransferRobot.kt.*"/>
-    <ignore regexp=".*OnlineLibraryRobot.kt.*"/>
-    <ignore regexp=".*ReaderRobot.kt.*"/>
-    <ignore regexp=".*SettingsRobot.kt.*"/>
-    <ignore regexp=".*TopLevelDestinationRobot.kt.*"/>
-    <ignore regexp=".*ZimHostRobot.kt.*"/>
-    <ignore regexp=".*SearchRobot.kt.*"/>
-    <ignore regexp=".*InitialDownloadRobot.kt.*"/>
-    <ignore regexp=".*DownloadRobot.kt.*"/>
+    <ignore regexp=".*BookmarksRobot.kt.*" />
+    <ignore regexp=".*DebugFunctions.kt.*" />
+    <ignore regexp=".*HistoryRobot.kt.*" />
+    <ignore regexp=".*IntroRobot.kt.*" />
+    <ignore regexp=".*LanguageRobot.kt.*" />
+    <ignore regexp=".*LibraryRobot.kt.*" />
+    <ignore regexp=".*LocalFileTransferRobot.kt.*" />
+    <ignore regexp=".*OnlineLibraryRobot.kt.*" />
+    <ignore regexp=".*ReaderRobot.kt.*" />
+    <ignore regexp=".*SettingsRobot.kt.*" />
+    <ignore regexp=".*TopLevelDestinationRobot.kt.*" />
+    <ignore regexp=".*ZimHostRobot.kt.*" />
+    <ignore regexp=".*SearchRobot.kt.*" />
+    <ignore regexp=".*InitialDownloadRobot.kt.*" />
+    <ignore regexp=".*DownloadRobot.kt.*" />
```

Comment:
This is not part of the issue. We should simplify the PR.

Reply:
done